packaging/darwin/Distribution.in

```diff
@@ -12,13 +12,6 @@
     <installation-check script="installCheck();"/>
     <script>
 function installCheck() {
-    if (9216000000 > system.sysctl('hw.memsize')) {
```

Comment:
should we make it it 4GB instead remove it completely?

Reply:
Looks like even 4G might not make sense if someone just want it to use for podman.